docs/design-v1.md

```diff
@@ -68,47 +76,25 @@ The cache service only accepts requests from other authorized services.
 Cache IDs are hashes that include the service token. Since service tokens are private, no two
 services can ever generate or guess the cache ID used by another service.
 
-### Python client
-
-In addition to a backend service, we can provide a simple Python client that makes it easy to upload large
-files using a streaming uploader/downloader.
-
-```py
-from kbase_caching_client import Cache
-
-cache = Cache({'service': 'MyService', 'token': 'xyz'})
-
-cache_params = {
-  'method': 'my_method',
-  'params': method_params
-}
-
-with cache(cache_params) as file:
-  # this block is only run when the cache is missing
-  file.write('xyz')
-  # When this block is completed, the file is cached, all IO is closed
-
-```
-
```

Comment:
No python client is planned then?

Reply:
I'd do it as an in-between thing if anyone wants it, otherwise I think the API is simple enough that it wouldn't be priority